### PR TITLE
[ONNX] add an utility function to check whether it's in the middle of onnx export or not

### PR DESCRIPTION
**Pull request by houseroad**

### CODE REVIEW

File: torch/onnx/__init__.py [hunk 1]
```diff
@@ -52,3 +53,8 @@ def _run_symbolic_function(*args, **kwargs):
 def _run_symbolic_method(*args, **kwargs):
     from torch.onnx import utils
     return utils._run_symbolic_method(*args, **kwargs)
+
+
+def is_in_onnx_export():
```
Comment (dzhulgakov) [inline torch/onnx/__init__.py]: add docstring to it?

**Review by dzhulgakov — APPROVED** [review]
A bit hacky, but I guess should work. I presume onnx is never called from multiple threads, so global is ok

File: torch/onnx/__init__.py (hunk 1 above)
Comment (houseroad) [reply torch/onnx/__init__.py]: Sure!

Comment (houseroad) [reply torch/onnx/__init__.py]: added the doc to onnx/utils.py. We probably should move all the doc to __init__.py

**Pull request closed without merging**